Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorHollowEffect.py

```diff
@@ -154,7 +154,7 @@ def onApply(self):
     kernelSizePixel = self.getKernelSizePixel()
     if shellMode == MEDIAL_SURFACE:
       # both erosion and dilation will be applied, so kernel size must be half on each side
-      kernelSizePixel = [kernelSizePixel[0]/2, kernelSizePixel[1]/2, kernelSizePixel[2]/2]
+      kernelSizePixel = [int(kernelSizePixel[0],2), int(kernelSizePixel[1],2), int(kernelSizePixel[2],2)]
```

Comment:
@lassoan  Would be great if you could review the changes associated with the Segment editor effects.

Reply:
The gist is that in `1/3` returns `0` in python 2 and returns `0.3333333` in python 3